features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/AppsAndKeys/ViewKeys.jsx

```diff
@@ -37,6 +36,7 @@ import DialogTitle from '@material-ui/core/DialogTitle';
 import Typography from '@material-ui/core/Typography';
 import TextField from '@material-ui/core/TextField';
 import { FormattedMessage, injectIntl } from 'react-intl';
+import Alert from '../Alert';
```

Comment:
We have to import `Alert` component [like this](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Applications/ApplicationFormHandler.jsx#L28)

Reply:
Earlier it was like that and it failed for the same reason.
 https://github.com/wso2/carbon-apimgt/pull/9197/commits/fb7e710d12656bd7157c0f4d1dd0634530cd9eac#diff-72ad92a507e0ec899f9c886e050af920L23

looks like Alert has imported twice in the code where Travis build is running